### DISCUSSION TITLE
JDK11 HttpClient Regression

### DISCUSSION THREAD

**Discussion by kwheat2064** [microsoft/openjdk#224] [Q&A] [answered]
Is there an ETA on release of a Microsoft OpenJDK11 containing the fix for the regression in the HTTPClient? (If this release is not impacted, great!)

See Link --> [https://github.com/adoptium/jdk11u/commit/ef5fff53ef1b047c2fca47047fe743689cc2734f](url)

**Comment by karianna** [comment, marked as answer]
We're actively working on this, I won't tempt fate with the test pipeline daemons on an exact ETA :-)

**Reply by kwheat2064** [reply]
Thanks @karianna!
